### PR TITLE
create rectangular padstack instance

### PR DESCRIPTION
**Pull request by svandenb-dev**

### CODE REVIEW

File: pyaedt/edb_core/padstack.py [hunk 1]
```diff
@@ -659,6 +660,8 @@ def create_padstack(
         endlayer : str, optional
             Ending layer. The default is ``None``, in which case the bottom
             is the ending layer.
+        pad_shape : str, optional
+            SHape of the pad. The default is ``"Circle``. Options are ``"Circle"`` and ``"Rectangle"``.
```
Comment (MaxJPRey) [inline pyaedt/edb_core/padstack.py:664]:
```suggestion
            Shape of the pad. The default is ``"Circle``. Options are ``"Circle"`` and ``"Rectangle"``.
```

File: _unittest/test_00_EDB.py [hunk 2]
```diff
@@ -2296,3 +2296,34 @@ def test_137_design_options(self):
...
+        pad = edb.core_padstack.create_padstack(
+            pad_shape="Rectangle",
+            padstackname="pad",
+            x_size="350um",
+            y_size="500um",
+            startlayer="top",
+            endlayer="top",
+            holediam=0,
+        )
+        pad_instance1 = edb.core_padstack.place_padstack(position=["-0.65mm", "-0.665mm"], definition_name="pad")
```
Comment (maxcapodi78) [inline _unittest/test_00_EDB.py:2314]: you should use new syntax:

```
edb.padstacks.place

```

**Review by maxcapodi78 — COMMENTED** [review]
comment on the methods and class used. Please don't use deprecated one

File: _unittest/test_00_EDB.py (hunk 2 above)
Comment (svandenb-dev) [reply _unittest/test_00_EDB.py]: Done thx

**Review by maxcapodi78 — APPROVED** [review]

**Pull request merged**